### ISSUE TITLE
Inellisense popup for XML documentation should honor Whitespace in ALL elements

### ISSUE THREAD

**Issue by smaillet** [dotnet/roslyn#74569]
**Version Used**: 
VS2022 17.10.4

**Steps to Reproduce**:
``` C#
/// <summary>Blah Blah Blah...</summary>
/// <remarks>
/// <para>Description of the following outline:</para>
/// 1. First point<br/>
/// 2. Second point<br/>
/// 3. Third point with indented sub point<br/>
///     1. Indented Sub point<br/>
/// 4. fourth point<br/>
/// </remarks>
```

**Expected Behavior**:
Inline whitespace (especially leading space for subpoint 3.1) would be honored and shown in IDE

**Actual Behavior**:
All leading whitespace is trimmed and left aligned in the popup (Looks terrible and harder to read). 

Creating Tables or outlined text is a Major PITA with doc comments. Ideally it would honor some form of Markdown, which looks and renders fine even if ignored as markdown (so long as whitespace is honored)

**Comment by chrisoverzero** [comment]
Have you tried using the [built-in features](https://learn.microsoft.com/en-us/dotnet/csharp/language-reference/xmldoc/recommended-tags#list) for lists and tables?

**Comment by smaillet** [comment]
Yes I have tried using the built in support. Besides being a horrible format that destroys readability of the comment. The existing list+list item are far too intrusive and don't handle the indented sub item anyway.

**Comment by CyrusNajmabadi** [comment]
Supporting 'br' seems trivial Sam.  Can you pop off a quick pr for this?
